### PR TITLE
Adding Week1 Blog Post

### PR DESCRIPTION
**Pull request by ganimtron-10**
Adding the blog post for the first week.
I have successfully generated the docs and here's the result

msedge_bqiBBcVV1M

msedge_gLf8wNVe65

### CODE REVIEW

File: docs/source/posts/2022/2022-06-08-week-1-praneeth.rst [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+This week we started with our first technical meeting in which the weekly tasks were assigned. So I had to start with some background or canvas and draw a line using mouse clicks.
+
+
+What did you do this week?
+--------------------------
+
+I started with a simple ``Panel2D`` (which is basically a movable rectangle on which we can add other UI elements) as a background and then assigned its mouse click callback to print “**Clicked!**” to verify the event triggering.
+
+Then I modified the event callback to create a ``Rectangle2D`` at that current mouse position(Now you would ask, Why ``Rectangle2D``?? We wanted to change the width of the line, which wasn't possible with the regular line). This would create a rectangle at that point but it was having static size.
```
Comment (xtanion) [inline docs/source/posts/2022/2022-06-08-week-1-praneeth.rst:18]: had instead of `was having`

**Review by xtanion — COMMENTED** [review]
Hello @ganimtron-10, Really nice blog. The doc is generating just fine. PTAL at my comments below.
Thanks.

**Comment by m-agour** [comment]
> We are waiting for your review @m-agour

@skoudoro I reviewed it when it got published and it was superb with no typos but forgot to comment that sorry, it was a crowded week.

**Review by skoudoro — APPROVED** [review]
great, thanks @ganimtron-10! merging!

**Pull request merged**